docs/current/spies.md

```diff
@@ -7,15 +7,16 @@
 ### What is a test spy?
 
 A test spy is a function that records arguments, return value, the value of
-`this` and exception thrown (if any) for all its calls. A test spy can be an
-anonymous function or it can wrap an existing function.
+`this` and exception thrown (if any) for all its calls. There are two types of spies:
+Some are anonymous functions, while others wrap previously existing methods.
```

Comment:
Either it exists or it does not exist. A previously existing method has been removed. 

I think this specific change is detrimental, but otherwise it seems fine.